config/samples/namespace-level-config.yaml

```diff
@@ -6,5 +6,6 @@ metadata:
   labels:
     numaplane.numaproj.io/config: namespace-level-config
 data:
+  # TODO-PROGRESSIVE: before the PROGRESSIVE strategy is implemented, users will only be able to choose "pause-and-drain". Afterwards, "progressive" should also be an option. Remove this comment line after implementing PROGRESSIVE strategy.
```

Comment:
You will find `TODO-PROGRESSIVE` comments in other files as well. These comments will need to be addressed when the PROGRESSIVE strategy is implemented.